### PR TITLE
catch error when failed to get podList in function failureTrap

### PR DESCRIPTION
**Pull request by hanxiaoshuai**
**What this PR does / why we need it**:
catch error when failed to get podList in function failureTrap
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by hanxiaoshuai** [comment]
cc @janetkuo Cloud you help to review this PR? Thanks

File: test/e2e/apps/deployment.go [hunk 1]
```diff
@@ -140,6 +140,10 @@ func failureTrap(c clientset.Interface, ns string) {
 		}
 		options := metav1.ListOptions{LabelSelector: selector.String()}
 		podList, err := c.CoreV1().Pods(rs.Namespace).List(options)
+		if err != nil {
+			framework.Logf("failed to get podList in namespace %s: %v", rs.Namespace, err)
```
Comment (tnozicka) [inline test/e2e/apps/deployment.go]: `Failed to list Pods in ...`?

Comment (hanxiaoshuai) [reply test/e2e/apps/deployment.go]: Done

**Review by tnozicka — APPROVED** [review]
/lgtm

**Comment by tnozicka** [comment]
/assign @janetkuo 
(for approval)

**Review by janetkuo — APPROVED** [review]

**Comment by janetkuo** [comment]
/approve

Thank you!

**Pull request merged**